etc/otel-collector-config.yaml

```diff
@@ -0,0 +1,36 @@
+receivers:
```

Comment:
I think we should reuse *this* config in `sdk-node` instead of duplicating it in both repos.

Reply:
Yeah I'll do that after merging